habitat_sim/utils/datasets_download.py

```diff
@@ -480,6 +485,11 @@ def main(args):
         action="store_true",
         help="If set, existing equivalent versions of any dataset found during download will be deleted automatically. Otherwise user will be prompted before overriding existing data.",
     )
+    arg_group2.add_argument(
+        "--no-replace",
+        action="store_true",
```

Comment:
store no replace in the same dest kwarg and have it default to None. That way its either True, False or None. See setup.py lto flag for an example.